proposals/0265-unlifted-datatypes.rst

```diff
@@ -126,10 +126,10 @@ data type explicitly *may* have levity polymorphic kind
 
 ::
 
-  data List a :: TYPE (BoxedRep LiftedRep) where
+  data List a :: TYPE (BoxedRep Lifted) where
     Nil :: List a
     Const :: a -> List a -> List a
-  data SList a :: TYPE (BoxedRep UnliftedRep) where
+  data SList a :: TYPE (BoxedRep Unlifted) where
     Nil :: SList a
     Const :: a -> SList a -> SList a
   data PList a :: TYPE (BoxedRep l) where
```

Comment:
Currently, `PNil` and `PCons` below have the following types (I'd add this review comment to the actual lines, but GitHub won't let me):

```hs
    PNil :: List a
    PCons :: a -> List a -> List a
```

Should these actually be like so?

```hs
    PNil :: PList a
    PCons :: a -> PList a -> PList a
```